lib/report/excel_export.rb

```diff
@@ -42,6 +42,60 @@ def generate_file
     p.to_stream().read()
   end
 
+  def generate_full_file
+    p = Axlsx::Package.new
+
+    img = File.expand_path('../image1.jpeg', __FILE__)
+
+    p.workbook.styles.fonts.first.name = 'Calibri'
+    p.workbook.add_worksheet(:name => "ChemOffice") do |sheet|
+      if @@sample_list.empty? || @@sample_list.first == nil
+        break
+      end
+
+      header = [""] + @@sample_list.first.attributes.keys
```

Comment:
uses `Sample.attribute_name` instead.
Check requirements. not all attributes should be exported